schemas/com.snowplowanalytics.snowplow.storage.rdbshredder/shredding_complete/jsonschema/1-0-0

```diff
@@ -0,0 +1,86 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for information about the shredded data, for loader to read",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow.storage.rdbshredder",
```

Comment:
In the code you used `com.snowplowanalytics.snowplow.storage.rdbloader`. I think that loader is better if we consider that shredder is a subpart of loader.

Reply:
I actually was thinking about that, but decided to go with shredder just because we already have one (unused) schema with this vendor. However, after thinking more, I agree that semantic meaning is more important than consistency with legacy schemas.